sdcm/cluster.py

```diff
@@ -1510,19 +1510,22 @@ def wait_for_nodes_up_and_normal(self, nodes):
 
     def get_scylla_version(self):
         if not self.nodes[0].scylla_version:
-            try:
-                result = self.nodes[0].remoter.run("rpm -q {}".format(self.nodes[0].scylla_pkg()))
-            except Exception as ex:
-                self.log.error('Failed getting scylla version: %s', ex)
-            else:
-                match = re.findall("scylla-(.*).el7.centos", result.stdout)
-                if match:
-                    scylla_version = match[0]
-                    self.log.info("Found ScyllaDB version: %s" % scylla_version)
-                    for node in self.nodes:
-                        node.scylla_version = scylla_version
+            version_commands = ["scylla --version", "rpm -q {}".format(self.nodes[0].scylla_pkg())]
+            for version_cmd in version_commands:
+                try:
+                    result = self.nodes[0].remoter.run(version_cmd)
+                except Exception as ex:
+                    self.log.error('Failed getting scylla version: %s', ex)
                 else:
-                    self.log.error("Unknown ScyllaDB version")
+                    match = re.match("(\d+[.]\d+[.]\w+).*", result.stdout)
```

Comment:
The new pattern doesn't work with output of `rpm -q`,  let's only use general `scylla --version`?

Reply:
```
>>> re.match("(\d+[.]\d+[.]\w+).*", '2018.1.2-0.20180527.c1f1556').group(0)
'2018.1.2-0.20180527.c1f1556'
>>> re.match("(\d+[.]\d+[.]\w+).*", '2018.1.2-0.20180527.c1f1556').group(1)
'2018.1.2'

 >>> print re.match("(\d+[.]\d+[.]\w+).*", 'scylla-enterprise-2018.1.2-0.20180527.c1f1556.el7.x86_64')
None
```